### PR TITLE
migrate cadvisor jobs to its own config file

### PR DESCRIPTION
**Pull request by krzyzacy**
also add @dashpole to OWNERS file

/assign @BenTheElder @dashpole

### CODE REVIEW

File: config/jobs/cadvisor/cadvisor.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180706-944ac61f8-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/kubernetes"
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - --root=/go/src
+        - --service-account=/etc/service-account/service-account.json
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --timeout=90
+        - --scenario=kubernetes_e2e
```
Comment (dashpole) [inline config/jobs/cadvisor/cadvisor.yaml]: where did all the extra args come from?

File: config/jobs/cadvisor/cadvisor.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20180510-809cd85b5
+      args:
+      - --repo=github.com/google/cadvisor
+      - --root=/go/src
+      - --timeout=30
+      - --scenario=canarypush
+      - --
+      - --file=deploy/canary/Dockerfile
```
Comment (dashpole) [inline config/jobs/cadvisor/cadvisor.yaml]: same here

File: config/jobs/cadvisor/cadvisor.yaml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      - --job=$(JOB_NAME)
+      - --root=/go/src
+      - --repo=k8s.io/kubernetes
+      - --repo=github.com/google/cadvisor=master
+      - --service-account=/etc/service-account/service-account.json
+      - --timeout=90
+      - --upload=gs://kubernetes-jenkins/logs
+      - --scenario=kubernetes_e2e
+      - --
+      - --deployment=node
```
Comment (dashpole) [inline config/jobs/cadvisor/cadvisor.yaml]: and here

**Comment by dashpole** [comment]
The first thing in kubernetes I am an owner of

File: jobs/config.json [hunk 4]
```diff
@@ -29,32 +29,6 @@
...
-      "--node-tests=true",
-      "--provider=gce",
-      "--test_args=--nodes=1--flakeAttempts=2",
-      "--timeout=10m"
-    ],
-    "scenario": "kubernetes_e2e",
-    "sigOwners": [
-      "sig-node"
-    ]
-  },
```
Comment (krzyzacy) [inline jobs/config.json]: @dashpole they come from here :-) forgot to commit the change

**Comment by krzyzacy** [comment]
what...
/retest

**Pull request merged**